DOC-1669: expanded explanation of what `state === true` actually entails for `obj` parameter. (#2438)
tinymce/tinymce-docs: modules/ROOT/pages/annotations.adoc

});
----

The `+obj+` parameter is only set if the `+state+` is true. And the `+state+` is

> * _true_ when the selection cursor moves in to an annotation; and

Reviewer: "moves in to" seems like weird phrasing?
```suggestion
* _true_ when the selection cursor moves into an annotation; and
```
